app/components/rdf-input-fields/rdf-date-input.hbs

```diff
@@ -0,0 +1,27 @@
+<AuLabel
+  for={{this.inputId}}
+  @error={{this.hasErrors}}
+  @warning={{this.hasWarnings}}
+  @required={{this.isRequired}}
+>{{this.title}}</AuLabel>
+<DateInput
+  @id={{this.inputId}}
+  @value={{this.date}}
+  @onChange={{this.onUpdate}}
+  @showErrorAsAlert={{false}}
```

Comment:
i would expect this one to also be restricted to the current period

Reply:
These restrictions are still set in the ttl as a validations *